dac2bids.py

```diff
@@ -293,8 +303,12 @@ def main():
     p.add_option('--sub', '-s', default=1, type="int", help="The subject number")
     p.add_option('--ses', '-e', default=1, type="int", help="The session number")
     p.add_option('--skipfmap', '-f', action="store_true", help="Skips fieldmaps.")
+    p.add_option('--taskname', '-t', default="task-unknown", help="The task name")    
 
+    global options 	# make options globally available
```

Comment:
Global is usually a bad idea because it allows `options` to be mutated from any place in the code. This can lead to subtle bugs. Also, if we were to import dac2bids and not call it as a main, then `options` would be used within `parse_protocols` without ever having been defined, leading to a `NameError`.